azure-pipelines.yml

```diff
@@ -0,0 +1,71 @@
+# Node.js with Angular
+# Build a Node.js project that uses Angular.
+# Add steps that analyze code, save build artifacts, deploy, and more:
+# https://docs.microsoft.com/azure/devops/pipelines/languages/javascript
+variables:
+  appDir: 'fakebook/'
+
+trigger:
+  branches:
+   include: 
+     - main
+  paths:
+    include:
+      - az-pipeline-angular.yml
```

Comment:
this path trigger excludes changes to the actual source code, while the (default) PR path trigger looks at any path. these should be consistent

Reply:
Updated to include more paths both in  triggers and pr